rotkehlchen/db/upgrades/v35_v36.py

```diff
@@ -542,6 +544,31 @@ def _add_okx(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_okx')
 
 
+def _upgrade_liquity_staking_events(write_cursor: 'DBCursor', conn: 'DBConnection') -> None:
+    log.debug('Enter _upgrade_liquity_staking_events')
+    with conn.read_ctx() as read_cursor:
+        read_cursor.execute(
+            'SELECT identifier, extra_data FROM history_events WHERE type=? AND subtype IN (?, ?) AND counterparty=?',  # noqa: E501
+            (
```

Comment:
why not filter where the extra data column is not `None` . The serialization method writes `NULL`  when the filter contains no information

Reply:
Yes, makes sense, thanks!